app/controllers/welcome_controller.rb

```diff
@@ -1,4 +1,16 @@
 class WelcomeController < ApplicationController
   def index
+
+    food_name = params[:food_name]
+
+    conn = Faraday.new(url: 'https://api.nal.usda.gov/fdc/v1/') do |faraday|
+      faraday.headers["api_key"] = Rails.application.credentials.fooddata[:key]
+    end
+    response = conn.get('foods/search', { query: food_name })
+
+    data = JSON.parse(response.body, symbolize_names: true)
```

Comment:
This is a good setup for getting the API connection going, but we do want to see you refactor into facades, services, and a poro, and then have tests for all 3 alongside your feature test.